server/html/views/home.html

```diff
@@ -385,5 +191,22 @@ <h3 class="mb-5 text-lg font-normal text-gray-500 dark:text-gray-400 whitespace-
       deleteHostBtn.disabled = true;
     }
   }
+
+  function onCloseModal() {
+    window.location = "/"
+  }
+
+  const monitoring_table = document.getElementById('monitoring');
+  const myRow = document.getElementById('myRow')
+  setInterval(async () => {
+    const response = await fetch('/update');
+    const data = await response.text();
+    if (monitoring_table) {
+      monitoring_table.innerHTML = "";
+      monitoring_table.innerHTML = data;
```

Comment:
Do we really need to set innerHTML to an empty string before setting it to data?